libr/core/agraph.c

```diff
@@ -4046,6 +4054,103 @@ static bool toggle_bb(RCore *core, ut64 addr) {
 	return false;
 }
 
+typedef struct r_canvas_location {
+	int x;
+	int y;
+} RPosition;
+
+static const char *strstr_xy(const char *p, const char *s, int *x, int *y) {
+	const char *d = strstr (p, s);
+	if (!d) {
+		return NULL;
+	}
+	const char *q;
+	int nl = 0;
+	int nc = 0;
+	for (q = p; q < d; q++) {
+		if (*q == '\n') {
+			nl++;
+			nc = 0;
+		} else {
+			nc++;
+		}
+	}
+	*x = nc;
+	*y = nl;
+	return d;
+}
+
+static char *old_word = NULL;
+static RList *word_list = NULL;
+static int word_nth = 0;
+
+static void nextword(RCore *core, RConsCanvas *can, const char *word) {
+	RListIter *iter;
+	if (word_list && !strcmp (word, old_word)) {
```

Comment:
old_word may be NULL and this can crash if you just press `\` a couple of times without first using `/`

Reply:
fixed